client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/tasks.tsx

```diff
@@ -149,10 +164,17 @@ export const launchpadFlowTasks: LaunchpadFlowTaskList = {
 	write: [
 		'setup_write',
 		'design_selected',
+		'verify_email',
 		'domain_upsell',
 		'first_post_published',
 		'site_launched',
 	],
-	videopress: [ 'videopress_setup', 'plan_selected', 'videopress_upload', 'videopress_launched' ],
+	videopress: [
+		'videopress_setup',
+		'plan_selected',
+		'verify_email',
+		'videopress_upload',
+		'videopress_launched',
+	],
 	sensei: [ 'sensei_setup', 'plan_selected', 'sensei_publish_first_course' ],
```

Comment:
I did not add the task to Sensei flow. It is a complex flow that handles a lot of logic separately and differently from other Launchpad flows. If we want to add this, it should likely be done as a follow up, and with strong input from the team working on this flow.